src/main/java/com/tstepnik/planner/service/TaskService.java

```diff
@@ -0,0 +1,83 @@
+package com.tstepnik.planner.service;
+
+import com.tstepnik.planner.domain.Importance;
+import com.tstepnik.planner.domain.Task;
+import com.tstepnik.planner.domain.User;
+import com.tstepnik.planner.exceptions.TaskNotFoundException;
+import com.tstepnik.planner.repository.TaskRepository;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TaskService {
+
+    private final Importance DEFAULT_IMPORTANCE = Importance.MINOR;
+
+    private final TaskRepository taskRepository;
+    private final AuthService authService;
+//TODO here add whole logic with if and exception and use loggedUser here!
```

Comment:
Remove todo if its already done. Also, for the future reference, add todos in a place where change should be made. I guess it makes sense for this one since it applies to whole class.

Reply:
I can i left it for remind me to keep logic there or i should rather remove it ?

Reply:
Then it should be regular comment and not todo